reports/helmcharts/helm-chart-versions.sh

```diff
@@ -130,21 +129,47 @@ for chart in $(echo "$charts" | jq -c '.[]'); do
   fi
 
   # Enhance document with additional information
+  helm_chart_name=$(echo "$chart" | jq -r '.name')
   uuid=$(uuidgen)
   created_on=$(date '+%Y-%m-%d %H:%M:%S')
+  id="${cluster_name}-${helm_chart_name}"
+
 
   document=$(echo "$chart" | jq --arg cluster_name "$cluster_name" \
                                 --arg verdict $verdict \
-                                --arg id "$uuid" \
+                                --arg id "$id" \
                                 --arg environment "$environment" \
                                 --arg created_on "$created_on" \
                                 --arg report_type "table" \
                                 --arg display_name "HELM Repositories" \
                                 --arg color_code $color_code '. + {id: $id, environment: $environment, createdOn: $created_on, lastUpdated: $created_on, displayName: $display_name, cluster: $cluster_name, verdict: $verdict, colorCode: $color_code, reportType: $report_type}')
 
   documents+=("$document")
+  
 done
 
+cosmosdb_account_name="sds-platform-version-reporter"
```

Comment:
You shouldn't be hardcoding anything related to the Azure resource itself as we plan to change this so it should be an input.